.github/workflows/dev.yml

```diff
@@ -1,57 +1,61 @@
 name: Development
 on: [pull_request, workflow_dispatch]
 
+env: 
+  # Override `DOCKER_IMAGE` variable in Makefile.docker 
+  # Thus, all Makefile target will use this docker image instead of building it.
```

Comment:
```suggestion
  # Override the `DOCKER_IMAGE` variable in Makefile.docker  
  # This makes all Makefile targets use the specified Docker image instead of building a new one.
```